packages/teleport/src/Databases/ButtonAdd/ButtonAdd.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { ButtonPrimary } from 'design';
+
+const docUrl = 'https://goteleport.com/docs/database-access/getting-started/';
+
+export default function ButtonAdd(props: Props) {
+  const { isEnterprise, canCreate, isLeafCluster, onClick } = props;
+
+  if (!isEnterprise) {
+    return (
+      <ButtonPrimary
+        width="240px"
+        onClick={() => null}
+        as="a"
+        target="_blank"
+        href={docUrl}
+        rel="noreferrer"
```

Comment:
just to keep this in mind for future: for links, if we dont use our `Link` component, we need to manually set `rel="noreferrer"` like we did here. https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer

In `Link` we set it auto: https://github.com/gravitational/webapps/blob/master/packages/design/src/Link/Link.jsx#L33